_data/library/jsfacades.yml

```diff
@@ -64,6 +64,10 @@
       url: https://github.com/coreyauger/scala-js-youtube
       desc: Type safe and Scala friendly library for the embedded YouTube player API
       dep: ''
+    - name: scala-js-momentjs
+      url: https://github.com/widok/scala-js-momentjs
+      desc: [Moment.js](http://momentjs.com/) facade
+      dep: libraryDependencies += "io.github.widok" % "scala-js-momentjs_sjs0.6_2.11" % "0.1.4"
```

Comment:
And the `%%%` formatting:

    dep: '"io.github.widok" %%% "scala-js-momentjs" % "0.1.4"'